sdgym/benchmark.py

```diff
@@ -764,13 +816,19 @@ def benchmark_single_table(
             limit the size of every table to 1,000 rows (randomly sampled) and the first 10
             columns.
         compute_quality_score (bool):
-            Whether or not to evaluate an overall quality score.
+            Whether or not to evaluate an overall quality score. Defaults to ``True``.
         compute_diagnostic_score (bool):
-            Whether or not to evaluate an overall diagnostic score.
-        sdmetrics (list[str]):
-            A list of the different SDMetrics to use. If you'd like to input specific parameters
+            Whether or not to evaluate an overall diagnostic score. Defaults to ``True``.
+        compute_privacy_score (bool):
+            Whether or not to evaluate an overall privacy score. Defaults to ``True``.
+        additional_sdmetrics (list[str]):
+            A list of the additional SDMetrics to use. If you'd like to input specific parameters
             into the metric, provide a tuple with the metric name followed by a dictionary of
             the parameters.
+        sdmetrics (list[str]):
+            **DEPRECATED**  A list of the different SDMetrics to use.
+            If you'd like to input specific parameters into the metric, provide a tuple with
+            the metric name followed by a dictionary of the parameters.
```

Comment:
I forgot to update the issue but I don't think we need to deprecate this anymore since we aren't going to run the new metric as part of this list.

Reply:
no, you can just leave it as sdmetrics. The only change here will be removing the NewRowSynthesis metrics from the default list and the default list altogether